config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
-  resources :users, only: [:new, :index, :create]
-
-  resources :static_pages, only: [:about, :index]
   root to: 'static_pages#home'
   get '/about', to: 'static_pages#about'
+  resources :static_pages, only: [:about, :index]
```

Comment:
you already have the `about` route as a path, what's `:index` for? if it shows all the playspaces, it should belong to playspaces resource perhaps?